cloud/resources/details.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Resource Details | Terramate Cloud
+description: Learn about details of a resource in Terramate Cloud.
+---
+
+# Resource Details
+
+On the Resource Details page, you can view the following details about the selected resource:
+- `Resource ID`: The unique identifier for the resource in Terramate Cloud.
```

Comment:
this is the Terraform id afaik.

Reply:
i wonder why it is empty in the screenshot. lets check this. or was it an initial deployment? then this might not yet be known. please verify.

Reply:
@mariux it's empty because I deleted it (tried to blur it) from the screenshot. Do you want it to be shown?